src/type-checker.cc

```diff
@@ -420,21 +420,11 @@ Result TypeChecker::OnBrTableTarget(Index depth) {
   if (br_table_sig_ == nullptr) {
     br_table_sig_ = &label_sig;
   } else {
-    if (features_.reference_types_enabled()) {
-      if (br_table_sig_->size() != label_sig.size()) {
-        result |= Result::Error;
-        PrintError("br_table labels have inconsistent arity: expected %" PRIzd
-                   " got %" PRIzd,
-                   br_table_sig_->size(), label_sig.size());
-      }
-    } else {
-      if (*br_table_sig_ != label_sig) {
-        result |= Result::Error;
-        PrintError(
-            "br_table labels have inconsistent types: expected %s, got %s",
-            TypesToString(*br_table_sig_).c_str(),
-            TypesToString(label_sig).c_str());
-      }
+    if (*br_table_sig_ != label_sig) {
+      result |= Result::Error;
+      PrintError("br_table labels have inconsistent types: expected %s, got %s",
+                 TypesToString(*br_table_sig_).c_str(),
+                 TypesToString(label_sig).c_str());
```

Comment:
Is this chunk unrelated?

Reply:
Yeah, you're right. I think I fixed it because it was caught when I updated the tests. I'll pull into a separate PR.